src/client/src/services/currencyPairs/currencyPairs.ts

```diff
@@ -28,7 +28,6 @@ export const [useCurrencyPairs, currencyPairs$] = bind(
       return result
     }, {} as Record<string, CurrencyPair>),
   ),
-  {},
 )
```

Comment:
@SHolleworth  - this is reversing your mod [here](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/pull/2202/files#diff-9b23749cb838b9b47fcb9646ac623980a71afa7a105d8a0db386fc8dab33aab7)
I don't know what this was added for, but it upsets the standalone spot tile .. can discuss later